### DISCUSSION TITLE
Users and permissions plugin not showing up on super admin

### DISCUSSION THREAD

**Discussion by VanjaRadovanovic** [strapi/strapi#8088] [Legacy Topics]
I'm having an issue when I create a new project and login with my super admin account I can't see or access users and permissions plugin but in marketplace I can see that plugin is installed. While installing and running project there were no errors. Any idea what's wrong?

**Comment by MattieBelt** [comment]
Hey @VanjaRadovanovic,

Checkout this (duplicate) issue: #8063

**Reply by MattieBelt** [reply]
It has been moved in `v3.1.6` (latest)
